projects/canopy/src/lib/content-scroll/content-scroll.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'lg-content-scroll',
+  styleUrls: ['./content-scroll.component.scss'],
+  templateUrl: './content-scroll.component.html',
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgContentSrollComponent {
+  @HostBinding('class.lg-content-scroll') class = true;
+  @Input() contentWidth: string = null;
+  @Input() contentHeight: string = null;
+  @Input() mobileFullContent = true;
```

Comment:
This was something I did not want to assume.  Yes, the project I've used this for requires full content view on small screens, but didn't want to restrict it.
Perhaps, from your second comment, a more acceptable approach would be to set the breakpoint (default to sm).  This would allow more control and adaptability to designs.